board/tests.py

```diff
@@ -55,4 +61,23 @@ def test_admin_board_connections(self):
     self.assertQuerysetEqual(u2.board_set.all(), [b1], ordered=False)
     self.assertQuerysetEqual(u3.board_set.all(), [b2], ordered=False)
     self.assertQuerysetEqual(Board.objects.filter(admin_users__id=1), [b1, b3], ordered=False)
-    self.assertQuerysetEqual(Board.objects.filter(admin_users__id=3), [b2], ordered=False)
\ No newline at end of file
+    self.assertQuerysetEqual(Board.objects.filter(admin_users__id=3), [b2], ordered=False)
+
+  @tag('core')
+  def test_post_creation_combinations(self):
+    """Make sure that creating posts in different combinations doesn't break anything."""
+    b1 = Board(title="foo", description="bar", bg=bytearray("biv", "utf-8"))
+    b1.save()
+
+    # just make sure that none of these throw an exception when creating
+    p1 = Post(associated_board=b1, message="post1")
+    p2 = Post(associated_board=b1, name="author2", photo=bytearray("post2", "utf-8"))
+    p3 = Post(associated_board=b1, photo=bytearray("post3", "utf-8"))
+    p4 = Post(associated_board=b1, name="author4", message="post4", photo=bytearray("pic4", "utf-8"))
+    p5 = Post(associated_board=b1, message="post5", photo=bytearray("pic5", "utf-8"))
+
+    p1.save()
+    p2.save()
+    p3.save()
+    p4.save()
+    p5.save()
```

Comment:
where's the actual `assertQuerysetEqual` part 🤨

Reply:
There is no `assertQuerysetEqual` part, that just checks to make sure you can create + save that with no exceptions...

I'm not too sure what else you would want as a test? Didn't you want me to create and test the `Post` creating with its combinations?

Reply:
I guess it was just making sure they all exist in the db